dark/index.html

```diff
@@ -232,8 +232,12 @@ <h3>ABOUT <span class="text-info">ME</span></h3>
             <h1>CONTACT ME</h1>
             <h3>CONTACT <span class="text-info">ME</span></h3>
             <p class="contact-text">
-                    📚 Education : St. Anthony's College, Kandy
+                    📚 Education : St. Anthony's College, Kandy.

```

Comment:
Remove this full stop from this @theotsuaneng 

```suggestion
                    📚 Education : St. Anthony's College, Kandy

```